File: solucoes/2011/nivel jr/fase 1/py/transporte.py
A,B,C = list(map(int, input().split()))
X,Y,Z = list(map(int, input().split()))

if(A <= X and B <= Y and C <= Z):
    qtd_containers = (X // A) * (Y // B) * (Z // C)
    print(int(qtd_containers))
else:
    print(0)

File: solucoes/2011/nivel jr/fase 1/js/transporte.js
var A,B,C 
var X,Y,Z 

scanf("%d%d%d", "A", "B", "C")
scanf("%d%d%d", "X", "Y", "Z")

if(A <= X && B <= Y && C <= Z){
    qtd_containers = Math.floor(X / A) * Math.floor(Y / B) * Math.floor(Z / C)
    printf("%d", qtd_containers)
}
else{
    printf("%d", 0)
}

